content/liveobjects/index.textile

```diff
@@ -8,21 +8,25 @@ product: liveobjects
 <p>LiveObjects is currently Experimental. Its features are still in development and subject to rapid change.</p>
 </aside>
 
-Ably LiveObjects enables effortless realtime synchronization of application state across multiple users and devices at any scale. LiveObjects provides a set of purpose-built APIs and data structures to handle the complexities of persisting and synchronizing state, freeing you to focus on building features instead of managing concurrency or conflict resolution.
+Ably LiveObjects provides a serverless, durable, and scalable way to create, update, and synchronize shared state across large numbers of connected clients at any scale.
 
-LiveObjects enables you to store data as "objects" on a channel. These objects are automatically synchronized in realtime across all connected clients, and any conflicts that arise from concurrent updates are seamlessly resolved in the background.
+LiveObjects provides a global, durable, and conflict-free shared data layer built on Ably's "global network":/docs/platform/architecture, so your application state stays perfectly synchronized in realtime without the need to build or manage complex infrastructure yourself.
```

Comment:
Should we refer to it as a "global platform" for consistency?

Reply:
Fixed in f122056f